notifications/service/service.go

```diff
@@ -0,0 +1,34 @@
+package service
+
+import (
+	"dont-slack-evil/apphome"
+	"log"
+	"os"
+
+	"github.com/slack-go/slack"
+)
+
+var slackBotUserOauthToken = os.Getenv("SLACK_BOT_USER_OAUTH_ACCESS_TOKEN")
+var slackBotUserApiClient = slack.New(slackBotUserOauthToken)
+
+func SendNotifications() error {
+
+	userIds, err := apphome.UsersWithBadQualityMessages()
+	if err != nil {
+		return err
+	}
+	for _, userId := range userIds {
+		conversationParameters := slack.OpenConversationParameters{
+			Users: []string{userId},
+		}
+		channel, _, _, conversationErr := slackBotUserApiClient.OpenConversation(&conversationParameters)
+		if conversationErr != nil {
+			log.Printf("Could not open conversation for user %v: %v", userId, conversationErr)
+		} else {
+			slackBotUserApiClient.PostMessage(channel.ID, slack.MsgOptionText("Test", false))
```

Comment:
Did you replace "connard" by "test" ? lol cf https://lesgarshack.slack.com/archives/CTVKMA481/p1583633871001600